src/components/composer/composer/composer_store.ts

```diff
@@ -68,6 +67,16 @@ export interface ComposerSelection {
 }
 
 export class ComposerStore extends SpreadsheetStore {
+  mutators = [
```

Comment:
any better name idea for `mutators` ?

Reply:
`modifiers` ? `state_mutators` ?
Actually, `mutators` is clear for me